db/migrations/20230104042228_create_plantsimg_table.sql

```diff
@@ -0,0 +1,11 @@
+-- migrate:up
+CREATE TABLE plantsimg (
```

Comment:
```suggestion
CREATE TABLE plant_images (
```
식물의 이미지들 (plant_images)로 지어주세요. 테이블명 가독성이 떨어집니다! 이부분도 저번 PR에서 드렸던 리뷰인데 반영이 안되었네요.